osmaxx-py/osmaxx/excerptexport/static/excerptexport/scripts/map.js

```diff
@@ -31,16 +31,10 @@
         };
 
         this.isSelectOptionSelectedAndExcerptOnMapInSyncWithInputFields = function(select, locationFilterBounds) {
-            if(select.value == "") {
+            if (select.value == "" || this.selectedExcerptGeoJson == null) {
```

Comment:
Is it still required to check `select.value` if it isn't used anymore in the remainder of this function?